pyccel/codegen/printing/cwrappercode.py

```diff
@@ -51,22 +52,47 @@ def get_cast_function(self, used_names, cast_type, from_variable, to_variable):
         if cast_type in self._cast_functions_dict:
             return self._cast_functions_dict[cast_type]
 
+        cast_function_name = self.get_new_name(used_names, cast_type)
         cast_function_arg = [from_variable]
         cast_function_result = [to_variable]
-        cast_function_ret = Return(cast_function_result)
-        cast_function_name = self.get_new_name(used_names, cast_type)
-        cast_function = CastFunction(cast_function_name, cast_type,
-                            cast_function_arg, cast_function_ret, cast_function_result)
+        cast_function_local_vars = []
+ 
+        #switch case of cast_type
+        if cast_type == 'pyint_to_bool':
+            cast_function_body = [Assign(cast_function_result[0], Bool(cast_function_arg[0]))]
+        elif cast_type == 'bool_to_pyobj':
+            cast_function_body = [If((Bool(cast_function_arg[0]),
+                [Assign(cast_function_result[0], Symbol('Py_True'))]),
```

Comment:
Can we use Constant or Variable instead of Symbol? Symbol is a sympy object and we tend to have problems when sympy objects still exist at the codegen step